syntax = "proto3";

service TestXAppService {
  rpc CheckServerStatus (StatusRequest) returns (StatusResponse);
  rpc StreamServerStatus (StatusRequest) returns (stream StatusResponse);
}

message StatusRequest {
  // This message can be empty for now
}

message StatusResponse {
  bool isRunning = 1;
  string message = 2;
}

